classes/Tools.php

```diff
@@ -3071,6 +3160,16 @@ public static function ZipTest($from_file)
         return $zip->open($from_file, ZipArchive::CHECKCONS) === true;
     }
 
+    /**
+     * @deprecated Deprecated since 1.7.0
+     *
+     * @return bool
+     */
+    public static function getSafeModeStatus()
```

Comment:
`getSafeModeStatus` has been removed in this PR https://github.com/PrestaShop/PrestaShop/pull/27957

Reply:
also ! Thx